### ISSUE TITLE
Package.loadSettings is deprecated.

### ISSUE THREAD

**Issue by kyoh86** [devongovett/atom-jade#36]
Store package settings files in the `settings/` directory instead of `scoped-properties/`
```
Package.loadSettings (/Applications/Atom.app/Contents/Resources/app/src/package.js:565:9)
<unknown> (/Applications/Atom.app/Contents/Resources/app/src/package.js:187:43)
```

**Comment by dwhoban** [comment]
dupe of #30

**Comment by LinusU** [comment]
Fixed in 0.4.0
